test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -66,19 +68,29 @@ const loopbackRedundantPollingMock = RequestMock()
   .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
   .respond(async (req, res) => {
     res.headers['content-type'] = 'application/json';
-    if (!firstPollCalledForRedundantPolling) {
-      firstPollCalledForRedundantPolling = true;
+    switch (loopbackRedundantPollingMockPollCount++) {
+    case -1:
+      res.statusCode = '200';
+      res.setBody(identifyWithUserVerificationLoopback);
+      break;
+    case 0:
```

Comment:
nit: indent the cases within the switch statement

Reply:
i actually prefer your way but the current linting rules want me to indent it this way